app/mixins/item_actions.js

```diff
@@ -212,32 +212,6 @@ export default Ember.Mixin.create(AsyncMixin, {
         this.get("actionTarget"),
         this.get("actionName")
       );
-    },
-
-    /**
-     * Unpack the requested quantity from a container (either box or pallet)
-     * and invoke any callback passed as argument.
-     * @param EmberObject container
-     * @param EmberObject item
-     * @param Integer quantity
-     * @param function callback
-     */
-    async unpack(container, item, quantity, callback) {
```

Comment:
I would suggest to keep this function
Reason: 
`unpack` always needs to be done to a location. Having the location logic inside a component/controller will be repetitive. And hence the location selection and unpacking logic moved to mixin, so that its easily pluggable to any component/controller.

If you need to return true/false, it would be better to return a `false` instead of doing just return
```javascript
const selectedLocation = await this.get(
        "locationService"
      ).userPickLocation();

      if (!selectedLocation) {
        return false;
      }
```